### ISSUE TITLE
Error on creating index

### ISSUE THREAD

**Issue by alireza2281** [Jeroen-G/Explorer#81]
Hello, after running `sail php artisan elastic:create` i'm getting this error

```
{"error":{"root_cause":[{"type":"parse_exception","reason":"key [settings] must be an object"}],"type":"parse_exception","reason":"key [settings] must be an object"},"status":400}
```
my explorer.php file content is:

```
<?php

declare(strict_types=1);

return [
    /*
     * There are different options for the connection. Since Explorer uses the Elasticsearch PHP SDK
     * under the hood, all the host configuration options of the SDK are applicable here. See
     * https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/configuration.html
     */
    'connection' => [
        'host' => 'es01',
        'port' => '9200',
        'scheme' => 'http',
    ],

    /**
     * An index may be defined on an Eloquent model or inline below. A more in depth explanation
     * of the mapping possibilities can be found in the documentation of Explorer's repository.
     */
    'indexes' => [
        \Modules\Contract\Entities\Contract::class,
        \Modules\Content\Entities\Blog::class
    ],

    /**
     * You may opt to keep the old indices after the alias is pointed to a new index.
     * A model is only using index aliases if it implements the Aliased interface.
     */
    'prune_old_aliases' => true,
];
```

and i have implemented Explorer "mappableAs" for both my models

**Comment by Jeroen-G** [comment]
Can you show me the mappableAs implementations?

**Comment by alireza2281** [comment]
Sure

The mappableAs of Blog model:

```
public function mappableAs(): array
    {
        return [
            'id' => 'keyword',
            'title' => 'text',
            'category' => 'text',
            'tag' => 'text',
            'excerpt' => 'text',
            'author' => 'text',
            'slug' => 'text',
            'created_at' => 'date',
        ];
    }
```

and mappableAs of contract mode:

```
public function mappableAs(): array
    {
        return [
            'id' => 'keyword',
            'name' => 'text',
            'description' => 'text',
            'created_at' => 'date',
        ];
    }
```

**Comment by Jeroen-G** [comment]
Hmm, that seems ok. What versions of Laravel, Laravel Scout and Explorer are you running? I will have to see if I can reproduce this.

**Comment by alireza2281** [comment]
OK, 

Laravel: `8.77.1` , Laravel Scout:  `9.4.2`, Explorer: `2.5` 

by the way i'm using Laravel sail for development and docker-compose.yml elastik part is like this:

```
es01:
      image: docker.elastic.co/elasticsearch/elasticsearch:7.9.3
      container_name: es01
      environment:
        - node.name=es01
        - cluster.name=es-docker-cluster
        - discovery.type=single-node
        - bootstrap.memory_lock=true
        - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
      ulimits:
        memlock:
          soft: -1
          hard: -1
      volumes:
        - data01:/usr/share/elasticsearch/data
      ports:
        - 9200:9200
      networks:
        - sail
kib01:
    image: docker.elastic.co/kibana/kibana:7.9.3
    container_name: kib01
    ports:
      - 5601:5601
    environment:
      ELASTICSEARCH_URL: http://es01:9200
      ELASTICSEARCH_HOSTS: http://es01:9200
    networks:
      - sail
```

and if it's necessary i'm using [Laravel-Modules](https://github.com/nWidart/laravel-modules) and these two models are in separate modules

**Comment by Jeroen-G** [comment]
Thanks, it looks the same as in the https://github.com/Jeroen-G/explorer-demo project

**Comment by Jeroen-G** [comment]
Ok, this is indeed a bug in 2.5 when a model does not have the IndexSettings interface implemented. I'll try to fix it as soon as I find the time!

**Comment by Jeroen-G** [comment]
Fixed in 2.5.1. Thanks for reporting!
